Sources/Datadog/DatadogConfiguration.swift

```diff
@@ -5,7 +5,11 @@
  */
 
 import Foundation
-import DatadogInternal
+
+@_exported import enum DatadogInternal.DatadogSite
+@_exported import struct DatadogInternal.AttributeKey
+@_exported import protocol DatadogInternal.AttributeValue
+@_exported import enum DatadogInternal.TrackingConsent
```

Comment:
I was thinking we will do it in `Example` app project - do it once (in `AppDelegate`?) and have these symbols re-exported to the entire app module 🤔💭. This could be minimal and isolated solution to our inconvenience.

Unlike what we seek for in `Example` app, our users don't need to have these types available for variables declaration, e.g.:
```swift
let trackingConent: TrackingConsent
```
They will only use it for variables assignment:
```swift
.set(trackingConsent:)
```
so we don't need to re-export their visibility.

Also, `@_exported` is discouraged by Swift maintainers, same as other [underscored attributes](https://github.com/apple/swift/blob/main/docs/ReferenceGuides/UnderscoredAttributes.md). Better to avoid it if possible 😉. Here, it's perfectly possible as the problem is only in our `Example` project. WDYT @maxep ?

Reply:
I think we only need the `TrackingConsent` in the Example, I will double check.

But it's very likely that users will need it as well, to implement any consent logic, e.g. storing in UserDefault or map the type with a Consent Provider. So we would probably want to expose it somehow, no?

I saw that the underscored attributes are discouraged but I don't see the risk with this one, other than semantic change. But to avoid any, the other option is having a wrapper exposed by the `Datadog` module:
```swift
public enum DatadogTrackingConsent {
    ...
    internal var _internal: DatatogInternal.TrackingConsent { get }
}
```
I will go with that option, if that' ok with you?

Reply:
Well, the wrapper has a larger impact than I thought :/ I will go with your suggestion to move forward 👍

Reply:
> But it's very likely that users will need it as well, to implement any consent logic, e.g. storing in UserDefault or map the type with a Consent Provider. So we would probably want to expose it somehow, no?

Could be 👌, but this is quite an assumption that we have no proof for. I'd say let's keep it simple for now - if one asks for it, we will think of solution. We could always add thin wrapper just for SDK configuration part (in `Datadog`) that maps into `DatadogInternal.TrackingConsent` right after configuration (with no other use than configuration it will have minimal to no impact).